tests/serverpod_test_server/test_e2e/service_protocol_test.dart

```diff
@@ -17,6 +17,18 @@ void main() {
     authenticationKeyManager: ServiceKeyManager('0', 'password'),
   );
 
+  group('Health metrics', () {
+    test('Fetch health metrics', () async {
+      // Fetch something far back, there should be no data.
+      var result = await serviceClient.insights.getHealthData(
+        DateTime(1976, 09, 10),
+        DateTime(1980, 04, 02),
+      );
+
+      expect(result.metrics.length, equals(0));
```

Comment:
Suggestion(Non blocking): Only assert that result is not null.

Since the length of `metrics` is not important for the test, we could just assert that the result is not null, ie. the fetch was successful.